README.md

```diff
@@ -7,9 +7,9 @@ All downloads are now located at our website that is located [here](https://chao
 Due to drama and various mass reports by unknown sources, the mod was taken down from [*CurseForge*](https://www.curseforge.com/minecraft/mc-mods) and cannot be re-uploaded from this point forward. Please do **not** harass **anyone** in the *CurseForge* moderation team or **any** competitor mod developer for what you think they did. They are humans and we are too, and we all make mistakes.
 
 ### Required Dependencies
-**1.16.5 (Forge):** [Forge v36.1.1 and up](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.16.5.html), [GeckoLib v3.0 and up](https://www.curseforge.com/minecraft/mc-mods/geckolib/files/all?filter-game-version=1738749986%3a70886)
+**1.16.5 (Forge):** [Forge v36.1.1 and up](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.16.5.html), [GeckoLib v3.0 and up](https://www.curseforge.com/minecraft/mc-mods/geckolib/files/all?filter-game-version=1738749986%3a70886) and [AttributeFix v10.1.4 and above](https://www.curseforge.com/minecraft/mc-mods/attributefix/files/all?filter-game-version=1738749986%3a70886)
 ### Recommended Dependencies
-[AttributeFix v10.0.1 and above](https://www.curseforge.com/minecraft/mc-mods/attributefix/files/all?filter-game-version=1738749986%3a70886)
+**1.16.5 (Forge):** [Just Enough Items, any recent version.](https://www.curseforge.com/minecraft/mc-mods/jei/files/all?filter-game-version=2020709689%3A8203), [Just Enough Resources, requires JEI, any recent version.](https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer/files/all?filter-game-version=2020709689%3A8203) [Stable version of JER recommended.)
```

Comment:
Maybe under the Recommended Dependencies section add a new section with mods with integration and one for chaos addons?
https://www.curseforge.com/minecraft/mc-mods/projecte
https://www.curseforge.com/minecraft/mc-mods/just-enough-effect-descriptions-jeed
https://www.curseforge.com/minecraft/mc-mods/jade
https://www.curseforge.com/minecraft/mc-mods/the-one-probe



https://revstudios.cf/mods/pure-chaos/
https://blackout03.github.io/mods/chaos-additions.html


and if it ever gets reviewed: 😐
https://modrinth.com/mod/spartanawakens